g/grpc/grpcio_ubi_9.3.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : grpcios
+# Version       : v1.64.0
+# Source repo   : https://github.com/grpc/grpc.git
+# Tested on     : UBI:9.3
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Bhagyashri Gaikwad <Bhagyashri.Gaikwad2@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=grpc
+PACKAGE_VERSION=${1:-v1.64.0}
+PYTHON_VERSION=${2:-3.9}
```

Comment:
No need to add `PYTHON_VERSION` variable and python installation instructions in build-scripts. Use just `python` and `pip`, rather using `pip3` or `python3` in your commands. Also avoid creating venvs in build-scripts.
For testing, you can create venv with desire python version in your ssh terminal and test your build-scripts.